model/database/metrics.yaml

```diff
@@ -133,3 +134,39 @@ groups:
     attributes:
       - ref: db.client.connection.pool.name
         requirement_level: required
+
+  - id: metric.db.cosmosdb.operation.request_charge
+    type: metric
+    metric_name: db.cosmosdb.operation.request_charge
+    brief: "Request Charge consumed by the operation"
+    instrument: histogram
+    unit: "{request_units}"
+    stability: experimental
+    extends: attributes.db.cosmosdb.minimal
+
+  - id: metric.db.cosmosdb.operation.max_item_count
+    type: metric
+    metric_name: db.cosmosdb.operation.max_item_count
```

Comment:
I don't think there is anything cosmos-specific about this metric - it could be called `db.query.limit` and would apply to any database. Cosmos can still reuse and customize it (e.g. add attributes if that's necessary).

Reply:
renamed it to` db.query.limit`